### PR TITLE
Require prefer: respond-async for super admin tools

### PR DESCRIPTION
**Pull request by codyebberson**

### CODE REVIEW

**Review by tonylee80 — APPROVED** [review]

**Comment by tonylee80** [comment]
Do we need to update the Super Admin Page included the header `Prefer: respond-async`?

https://github.com/medplum/medplum/blob/191f8c9399a6199d9f44d44711a5f40f26fe80be/packages/app/src/admin/SuperAdminPage.tsx

**Comment by codyebberson** [comment]
@tonylee80 

> Do we need to update the Super Admin Page included the header `Prefer: respond-async`?

Yep!  Sorry, didn't put this in draft state.  I'm adding that.

**Comment by codyebberson** [comment]
image

image

File: packages/core/src/client.ts [hunk 1]
```diff
@@ -2306,21 +2313,41 @@ export class MedplumClient extends EventTarget {
...
+  }
+
+  /**
+   * Starts an async request following the FHIR "Asynchronous Request Pattern".
+   * See: https://hl7.org/fhir/r4/async.html
+   * @param url The URL to request.
+   * @param options Optional fetch options.
+   * @returns The response body.
+   */
+  async startAsyncRequest<T>(url: string, options: RequestInit = {}): Promise<T> {
```
Comment (tonylee80) [inline packages/core/src/client.ts:2326]: Nice!

**Review by tonylee80 — APPROVED** [review]
Looks good!

image

**Pull request merged**